modules/nuxt-swaggerjsonapi.yml

```diff
@@ -0,0 +1,14 @@
+name: nuxt-swaggerjsonapi
+description: 'Efficient development, fast copy and paste use for Nuxt.js'
+repo: 516310460/nuxt-swaggerjsonapi
+npm: nuxt-swaggerjsonapi
+icon: nuxt-swaggerjsonapi.svg
```

Comment:
You define your icon but not added into the project ?

Reply:
I left the icon empty

Reply:
I added my own icon